hbase-thrift/src/test/java/org/apache/hadoop/hbase/thrift2/TestThriftHBaseServiceHandler.java

```diff
@@ -1767,6 +1769,12 @@ public void testSlowLogResponses() throws Exception {
     clearedResponses.forEach(Assert::assertTrue);
     TLogQueryFilter tLogQueryFilter = new TLogQueryFilter();
     tLogQueryFilter.setLimit(15);
+    Assert.assertEquals(tLogQueryFilter.getFilterByOperator(), TFilterByOperator.OR);
+    LogQueryFilter logQueryFilter = ThriftUtilities.getSlowLogQueryFromThrift(tLogQueryFilter);
```

Comment:
Umm, if we came this far, why not make an RPC and validate the result set too? :-)

Reply:
Ah I see, yes that is quite valid point :)
However, the issue with testing this way is: For our case, we don't have any Add/Update API, we just have Get API, and the event that trigger Add event is something not in client test's control (only at RpcServer level, we have control to identify if an RPC call was slow/large by nature).